packages/examples/src/huge.ts

```diff
@@ -0,0 +1,916 @@
+import { registerExamples } from './register';
```

Comment:
This example breaks the layout of the example page. I think this is the case because of very long strings. Can we wrap them, perhaps?